build-kite/vm_scripts/start-agent.sh

```diff
@@ -34,12 +34,16 @@ $AS_AGENT git config --global user.email "rich.fitzjohn+vimc@gmail.com"
 $AS_AGENT git config --global user.name "vimc-robot"
 $AS_AGENT git config --global push.default simple
 
+# Add webhook URLs to environment
+HINT_WEBHOOK=$(vault read -field=url secret/hint/flow-webhook)
```

Comment:
is this a hint or a hintr hook? If the hook will be used by both hint and hintr this name is fine

Can we make the envvar more descriptive (HINT_TEAMS_WEBOOK or HINT_WEBHOOK_TEAMS).

Reply:
Thanks, updated that. Yeah expecting this webhook to be used for every build that wants to post to ci-hint channel